api/scanner/exif/exif.go

```diff
@@ -31,7 +31,7 @@ func SaveEXIF(tx *gorm.DB, media *models.Media) (*models.MediaEXIF, error) {
 		}
 	}
 
-	var parser exifParser = &internalExifParser{}
+	var parser exifParser = &externalExifParser{}
```

Comment:
I see that you've deleted the internalExifParser, I think it would be best to keep that one and on startup check if `exiftools` is installed and set `parser` to the external or internal parser based on that.

Reply:
Good point, didn't consider that :) I'll keep the internal parser.

Reply:
Perfect :)